training/README.md

```diff
@@ -2,4 +2,4 @@
 
 This folder contains current NN training code. Make sure to include all files to be transferred in `train.submit`! Also, check the graph and model saving lines in `train.py` if you want to convert it to a notebook or vice versa, otherwise they might not save correctly.
 
-Note: The current training set is at `/scratch365/klannon/dnnlikelihood/likelihood_data_no_delta.npz`.
\ No newline at end of file
+Note: The current training set is at `/scratch365/klannon/dnnlikelihood/likelihood_data_no_delta.npz`. A copy can also be found on CurateND.
```

Comment:
idk how CurateND works but perhaps add a link to where it is?

Reply:
Done